service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/AwaitCloneAllResourcesFlightStep.java

```diff
@@ -34,8 +32,6 @@ public class AwaitCloneAllResourcesFlightStep implements Step {
 
   public AwaitCloneAllResourcesFlightStep() {}
 
-  private static final int AWAIT_CLONE_RESOURCES_POLL_CYCLES = 720;
```

Comment:
This would poll for a total of 12 minutes because we polled every second. We now poll for an hour maximum, using the constants defined in https://github.com/DataBiosphere/terra-workspace-manager/pull/1688/files#diff-af300abcc49577c64d79724ef7af4288bbe77f0e6f5ed6c4e42e2a9a323379adR32.

Reply:
👍 an hour seems like a better idea, I'm a bit surprised it was 12 minutes.